packages/components/Tabs/src/TabsItem.tsx

```diff
@@ -39,24 +38,32 @@ export const TabsItem = compose<TabsItemType>({
     // Grabs the context information from Tabs (currently selected TabsItem and client's onTabsClick callback).
     const info = React.useContext(TabsContext);
 
-    /* There's a bug where the user callback is being called multiple times on one click.
-    We check that there is an actual change in selection before forwarding the message to the user callback
-    so that they aren't notified more than once for each click. */
-    const changeSelection = () => {
-      if (itemKey != info.selectedKey) {
+    const [focusState, setFocusState] = React.useState({
+      focused: false,
+    });
+
+    const changeSelection = React.useCallback(() => {
+      componentRef.current.focus();
```

Comment:
`componentRef?.current?.focus()` is the safest way to do this. It checks if it's undefined first. The way it's currently implemented, if `componentRef == nullptr || .current == nullptr`, it will cause an error